index.test-d.ts

```diff
@@ -0,0 +1,20 @@
+import {expectType} from 'tsd-check';
+import {default as callsites, CallSite} from '.';
```

Comment:
Does this not work in TS?

```suggestion
import callsites, {CallSite} from '.';
```